installation/linux/ubuntu.md

```diff
@@ -1,6 +1,6 @@
 # Ubuntu
 
-Fluent Bit is distributed as **fluent-bit** package and is available for the latest stable Ubuntu system: Focal Fossa.
+Fluent Bit is distributed as **fluent-bit** package and is available for the latest stable Ubuntu system: Jelly Jellyfish.
```

Comment:
it's Jammy Jellyfish.

Reply:
I checked this using markdown live inspector
![ubuntu22](https://user-images.githubusercontent.com/11384973/186030114-a3777ced-ec3d-4ad6-bbc4-e2d9a989e73b.gif)